JavaScript/JavaScript.sublime-syntax

```diff
@@ -109,8 +109,11 @@ variables:
 
 contexts:
   main:
-    - include: comments-top-level
+    - meta_include_prototype: false # don't match comments before shebang
+    - match: ''
+      push: [javascript, shebang]
 
+  javascript:
```

Comment:
Maybe just `script` for the name of this context? Might be less confusing in the inherited syntaxes.